src/client/components/AvailableReservations.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from "react";
+
+function AvailableReservations({ mealId }) {
```

Comment:
I would question if it needs to be a separate component, but otherwise well done!